packages/system/src/Item/api/hooks/useServiceItems/useServiceItems.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery, useQueryParams } from '@openmsupply-client/common';
+import { useItemApi } from './../useItemApi';
+
+export const useServiceItems = () => {
+  const api = useItemApi();
+  const queryParams = useQueryParams({ initialSortBy: { key: 'name' } });
+
+  return useQuery(api.keys.paramList(queryParams), () =>
+    api.get.serviceItems(queryParams)
+  );
```

Comment:
Maybe I should have just used the same api calls with filters. However I think the queries for stock items and service items will change in the future for different reasons and will cause headaches

Reply:
The separation is helpful - some with pagination, some not, different fragments etc